### PR TITLE
Results framework UI changes for EUTF

### LINKED ISSUE #2982: Modify results framework to better suit the EUTF workflow
The EUTF does not use reporting periods as a typical RSR results framework user.  The UI for EUTF can be modified as below, to suit their needs

- Remove/hide the ‘Needs Reporting’ tab 
- Remove/hide the ‘Approved’ tab
- Remove/hide the ‘Filter Period’ menu 
- Remove/hide the ‘Lock’ and ‘Unlock’ buttons

### PR DESCRIPTION
**Pull request by punchagan**
This branch is based off of #2940, and needs to be merged after that is merged. 

Closes #2982

- [ ] Test plan | Unit test | Integration test
- [ ] Copyright header
- [ ] Code formatting
- [ ] Documentation
- [ ] Change log entry

### CODE REVIEW

**Comment by zzgvh** [comment]
Both this and the bulk approval PR look fine, but are all EUTF projects' periods unlocked? Otherwise they just lost the tools to manage that...

**Comment by punchagan** [comment]
@zzgvh Ah, very good catch!  Also, given that by default a newly created indicator period is locked, there needs to be some way to unlock these periods. Any ideas, @zzgvh @AkvoAnt ?

File: akvo/rsr/management/commands/lock_unlock_periods.py [hunk 1]
```diff
@@ -21,28 +21,40 @@ class Command(BaseCommand):
         make_option('-k', '--keyword',
```
Comment (zzgvh) [inline akvo/rsr/management/commands/lock_unlock_periods.py]: I can't get keywords to work. They can be more than one word and even if i wrap them in " only the first token is used: ```./manage.sh lock_unlock_periods unlock -k "EU Trust Fund - North of Africa"``` results in the following debug output:
```
(Pdb) options
{'keyword': 'EU', 'settings': None, 'pythonpath': None, 'verbosity': u'1', 'traceback': None, 'no_color': False, 'organisation': None}
```

Comment (punchagan) [reply akvo/rsr/management/commands/lock_unlock_periods.py]: There are multiple problems here. 

1. The `manage.sh` script does some bash magic, and doesn't handle the args with spaces very well. Escape the spaces, or use `manage.py` from the vagrant box. 
2. They keyword you were trying to use has a unicode hyphen, it looks like `\u2013`.  Try using `'EU Trust Fund - Sahel and Lake Chad'`

File: akvo/rsr/static/scripts-src/my-results/components/FilterBar.jsx [hunk 2]
```diff
@@ -5,31 +5,32 @@
    < http://www.gnu.org/licenses/agpl.html >.
```
[the remark this answers is not in the document]
Comment (punchagan) [reply akvo/rsr/static/scripts-src/my-results/components/FilterBar.jsx]: I agree.  As a work around for this time, I ran prettier on all the files changed in this PR, in a separate commit first.  All other commits should be minimal changes.

**Comment by punchagan** [comment]
@zzgvh the points you raise about workflow annoyances are quite valid. Let's have a discussion on this and decide what route to take.

**Comment by punchagan** [comment]
Too old, and probably no longer needed.

**Pull request closed without merging**